pets/meupet/locale/pt_BR/LC_MESSAGES/django.po

```diff
@@ -2,250 +2,289 @@
+"É preto e gorduchinho, bastante arisco, desapareceu perto da escola no "
+"centro.\n"
 "Tem uma pequena falha no pelo da calda."
 
-#: pets/meupet/forms.py:54
+#: meupet/forms.py:55
 msgid "Image is larger than the maximum size of 8MB"
 msgstr "Imagem é maior do que o tamanho máximo de 8MB"
 
-#: pets/meupet/forms.py:62 pets/meupet/templates/meupet/_pet_form.html:44
-#: pets/meupet/templates/meupet/pet_detail.html:83
+#: meupet/forms.py:63 meupet/templates/meupet/_pet_form.html:44
+#: meupet/templates/meupet/pet_detail.html:83
 msgid "City"
 msgstr "Cidade"
 
-#: pets/meupet/forms.py:63 pets/meupet/templates/meupet/_pet_form.html:62
+#: meupet/forms.py:64 meupet/templates/meupet/_pet_form.html:62
 msgid "Kind"
 msgstr "Espécie"
 
-#: pets/meupet/forms.py:64 pets/meupet/templates/meupet/_pet_form.html:71
-#: pets/meupet/templates/meupet/pet_detail.html:89
-#: pets/meupet/templates/meupet/poster.html:37
+#: meupet/forms.py:65 meupet/templates/meupet/_pet_form.html:71
+#: meupet/templates/meupet/pet_detail.html:89
+#: meupet/templates/meupet/poster.html:37
 msgid "Size"
 msgstr "Tamanho"
 
-#: pets/meupet/forms.py:65 pets/meupet/templates/meupet/_pet_form.html:7
+#: meupet/forms.py:66 meupet/templates/meupet/_pet_form.html:7
 msgid "Status"
 msgstr "Situação"
 
-#: pets/meupet/forms.py:66 pets/meupet/templates/meupet/_pet_form.html:80
-#: pets/meupet/templates/meupet/pet_detail.html:96
+#: meupet/forms.py:67 meupet/templates/meupet/_pet_form.html:80
+#: meupet/templates/meupet/pet_detail.html:96
 msgid "Sex"
 msgstr "Sexo"
 
-#: pets/meupet/forms.py:78
+#: meupet/forms.py:79
 msgid "You must select at least one filter"
 msgstr "Você precisa selecionar pelo menos um filtro"
 
-#: pets/meupet/models.py:70
+#: meupet/models.py:86
 msgid "Female"
 msgstr "Fêmea"
 
-#: pets/meupet/models.py:71
+#: meupet/models.py:87
 msgid "Male"
 msgstr "Macho"
 
-#: pets/meupet/models.py:77
+#: meupet/models.py:93
 msgid "Small"
 msgstr "Pequeno"
 
-#: pets/meupet/models.py:78
+#: meupet/models.py:94
 msgid "Medium"
 msgstr "Médio"
 
-#: pets/meupet/models.py:79
+#: meupet/models.py:95
 msgid "Large"
 msgstr "Grande"
 
-#: pets/meupet/models.py:86
+#: meupet/models.py:102
 msgid "Missing"
 msgstr "Desaparecido"
 
-#: pets/meupet/models.py:87 pets/meupet/templates/meupet/_sidemenu.html:12
-#: pets/meupet/views.py:84
+#: meupet/models.py:103 meupet/templates/meupet/_sidemenu.html:12
+#: meupet/views.py:84
 msgid "For Adoption"
 msgstr "Para Adoção"
 
-#: pets/meupet/models.py:88
+#: meupet/models.py:104
 msgid "Adopted"
 msgstr "Adotado"
 
-#: pets/meupet/models.py:89
+#: meupet/models.py:105
 msgid "Found"
 msgstr "Encontrado"
 
-#: pets/meupet/models.py:106 pets/meupet/templates/meupet/pet_detail.html:65
+#: meupet/models.py:122 meupet/templates/meupet/pet_detail.html:65
 msgid "Maximum image size is 8MB"
 msgstr "Tamanho máximo da imagem é 8MB"
 
-#: pets/meupet/templates/meupet/_pet_form.html:16
+#: meupet/services.py:15
+msgid "Update pet registration"
+msgstr ""
```

Comment:
Yes, I'll delay the translations until the others parts are ready.
I'll add this observation to the PR comment, thanks @cuducos! :)

Could you try to find some errors in the English strings?
You know, your English should be better than mine.